strawman-Array.prototype.last.md

```diff
@@ -0,0 +1,62 @@
+# Array.prototype.last
+A proposal for an ECMAScript native method
+
+Code
+---
+
+    //everything in 6 lines
+    Array.prototype.last = function () {
```

Comment:
The benefit of what [/u/defproc](http://www.reddit.com/user/defproc) suggested is that now, instead of just getting the last element, you can also supply a predicate to get the last element that fits a criteria.